src/crypto.rs

```diff
@@ -43,18 +43,37 @@ pub fn name(public_key: &PublicKey) -> XorName {
     XorName(public_key.to_bytes())
 }
 
-/// Construct a random `Keypair`
-pub fn gen_keypair() -> Keypair {
-    Keypair::generate(&mut rand::thread_rng())
+/// Construct a random `Keypair` whose name's last byte represents the targeted age.
```

Comment:
Should this not be part of the gen keypair in a range?

Reply:
`gen keypair in a range` is about the `address`